File: frontend/src/app/JobOffer/joboffer.ts
export class JobOffer{
  id_job_offer: Number;
  id_proffessional: Number;
  title: String;
  body: String;
  applications: Number;

  constructor(id_proffessional: Number, title: String, body: String){
    this.id_job_offer = 0;
    this.id_proffessional = id_proffessional;
    this.title = title;
    this.body = body;
    this.applications=-1;
  }
}

File: LinkeDIn/backend/src/main/java/com/project/linkedin/JobOffer/JobOffer.java
package com.project.linkedin.JobOffer;

import lombok.Data;

import javax.persistence.*;


@Data
@Entity
public class JobOffer {

    private @Id @GeneratedValue(strategy = GenerationType.SEQUENCE) Long id_job_offer;
    private Long id_proffessional;
    private String title;
    private String body;

    public JobOffer() {
    }

    public JobOffer(Long id_proffessional,String title,String body) {

        this.id_proffessional = id_proffessional;
        this.title = title;
        this.body = body;

    }


}
